deegree maven plugin does clean up deegree-compliance-tests directory

This leads to growing size of the deegree-compliance-tests web app.

Workaround is available here: https://github.com/deegree/deegree3/compare/master...lat-lon:cleanPluginComplianceTests

Testfailures are caused by a change in https://github.com/deegree/deegree3/pull/1058/files#diff-41768028a4f464394c40f29346f004f1121204b40501c7e168d6ee04fa592d98
Adds: `graph.removeNode( id )` 
This removes nodes from the graph while other (dependents of the resource to destroy) are accessed.

First try: 
 1. Collecting all resources to remove
 2. Remove collected resources

This leads to a NPE in org.deegree.workspace.SimpleSqlFeatureStoreTest#testDestroyInitializeSingle():

```
org.deegree.workspace.ResourceInitException: Unable to build resource FeatureStoreProvider:simplesql-ok: null

	at org.deegree.workspace.standard.DefaultWorkspace.init(DefaultWorkspace.java:398)
	at org.deegree.workspace.SimpleSqlFeatureStoreTest.testDestroyInitializeSingle(SimpleSqlFeatureStoreTest.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
...
Caused by: java.lang.NullPointerException
	at org.deegree.feature.persistence.simplesql.DbFeatureUtils.determineFeatureType(DbFeatureUtils.java:110)
	at org.deegree.feature.persistence.simplesql.SimpleSQLFeatureStore.init(SimpleSQLFeatureStore.java:419)
	at org.deegree.workspace.standard.DefaultWorkspace.init(DefaultWorkspace.java:388)
	... 25 more
```

It seems that the the dependent JDBCConnection is not Initialised as expected.

--- a/deegree-tests/deegree-compliance-tests/pom.xml
+++ b/deegree-tests/deegree-compliance-tests/pom.xml
@@ -49,6 +49,20 @@
 
     <plugins>
       <plugin>
+        <artifactId>maven-clean-plugin</artifactId>
+        <version>3.1.0</version>
+        <configuration>
+          <filesets>
+            <fileset>
+              <directory>src/main/webapp</directory>
+            </fileset>
+            <fileset>
+              <directory>src/main/workspace</directory>
+            </fileset>
+          </filesets>
+        </configuration>
+      </plugin>
+      <plugin>
         <groupId>org.deegree</groupId>
         <artifactId>deegree-maven-plugin</artifactId>
         <executions>

--- a/deegree-tests/deegree-compliance-tests/pom.xml
+++ b/deegree-tests/deegree-compliance-tests/pom.xml
@@ -54,10 +54,10 @@
         <configuration>
           <filesets>
             <fileset>
-              <directory>src/main/webapp</directory>
+              <directory>${project.basedir}/src/main/webapp</directory>
             </fileset>
             <fileset>
-              <directory>src/main/workspace</directory>
+              <directory>${project.basedir}/src/main/workspace</directory>
             </fileset>
           </filesets>
         </configuration>

